### PR TITLE
[java] Remove DsoJson

### PR DESCRIPTION
**Pull request by wargio**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

- Removed DsoJson and using PJ
- Removed java/main.c
- Refactoring slightly the code to simplify it.

### CODE REVIEW

File: shlr/java/class.c [hunk 1]
```diff
@@ -726,97 +665,135 @@ RZ_API DsoJsonObj *rz_bin_java_get_class_info_json(RzBinJavaObj *bin) {
...
-		dso_json_dict_insert_str_key_str(class_info_dict, "name", class_->name);
-
-		if (!class_->super) {
-			DsoJsonObj *str = dso_json_str_new();
-			dso_json_dict_insert_str_key_obj(class_info_dict, "super", str);
-			dso_json_str_free(str);
-		} else {
-			dso_json_dict_insert_str_key_str(class_info_dict, "super", class_->super);
-		}
+		pj_kb(j, "access_flags", class_->visibility);
```
Comment (p371jah) [inline shlr/java/class.c]: `pj_ki`?

Comment (wargio) [reply shlr/java/class.c]: yes, i noticed i made a mistake. pj_kN should be better.

**Review by XVilka — APPROVED** [review]
Clear improvement in both code and the output. We will need more Java tests in the future though.

**Review by Maijin — APPROVED** [review]

**Comment by Maijin** [comment]
Solid!

**Pull request merged**